l10n_nl_bsn/models/res_partner.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# © 2016 ONESTEiN BV (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
+from openerp.tools.translate import _
+from stdnum.nl import bsn
```

Comment:
Please use an import that does not fail when the module is not available.
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md , External dependencies